packages/core/src/routes/resource.ts

```diff
@@ -75,6 +77,8 @@ export default function resourceRoutes<T extends AuthedRouter>(
     '/resources',
     koaGuard({
       body: Resources.createGuard.omit({ id: true }),
+      response: Resources.guard.merge(object({ scopes: Scopes.guard.array().optional() })),
```

Comment:
The `object` seems redundant.
```suggestion
      response: Resources.guard.merge({ scopes: Scopes.guard.array().optional() }),
```

Reply:
updated